tests/test_esm_runscripts/test_filedicts.py

```diff
@@ -155,3 +182,49 @@ def test_mv(fs):
         "/work/ollie/pgierz/some_exp/run_20010101-20010101/work/unit.24"
     )
 
+
+# ===
+# Tests for SimulationFile.ln() method
+# + check if source exists
+# + check if destination is written
+# + check if destination already exists or a symlink
+# + check for self pointing
+# + check if source path does not exist
+# + check if destination path does not exist
+# ===
+def test_ln_iscallable(simulation_file):
```

Comment:
I do not understand why this test is necessary.

Reply:
We can remove that actually. Since I write the test first and then the code as a part of the red->green->refactor cycle, I test first if the function is callable.